pkg/elementalcli/elementalcli.go

```diff
@@ -30,10 +30,16 @@ func Run(conf map[string]interface{}) error {
 	ev := mapToEnv("ELEMENTAL_INSTALL_", conf)
 
 	installerOpts := []string{"elemental"}
-	// Do we really need this? We already would set ELEMENTAL_DEBUG env var
-	if conf["debug"] == true {
+	// There are no env var bindings in elemental-cli for elemental root options
+	// so root flags should be passed within the command line
+	debug, ok := conf["debug"].(bool)
+	if ok && debug {
 		installerOpts = append(installerOpts, "--debug")
 	}
+	configDir, ok := conf["config-dir"].(string)
+	if ok && configDir != "" {
+		installerOpts = append(installerOpts, "--config-dir", configDir)
+	}
```

Comment:
I am wondering about the idea of setting a default here, so if `configDir` is unset default it to `/run/initramfs/live/elemental` so, in that case, any ISO could including an `/elemental/config.yaml` would fetch it by default, without having to include the `config-dir: "/run/initramfs/live/elemental"` flag in install which is not really obvious.

What do you think?

Reply:
I would probably err on the side of not adding another default, but I don't have a strong opinion either way..